src/fswAlgorithms/attGuidance/flybyPoint/flybyPoint.h

```diff
@@ -32,6 +33,7 @@ class FlybyPoint : public SysModel {
     ReadFunctor<NavTransMsgPayload> filterInMsg;              //!< input msg relative position w.r.t. asteroid
     ReadFunctor<EphemerisMsgPayload> asteroidEphemerisInMsg;  //!< input asteroid ephemeris msg
     Message<AttRefMsgPayload> attRefOutMsg;                   //!< Attitude reference output message
+    Message<FlybyDiagnosisMsgPayload> flybyDiagnosisOutMsg;
```

Comment:
:large_orange_diamond: Probably worth adding a comment on this public class variable.